providers/platform.yaml

```diff
@@ -0,0 +1,41 @@
+#ddev-generated
+
+auth_command:
+  command: |
+    set -eu -o pipefail
+    if [ -z "${PLATFORMSH_CLI_TOKEN:-}" ]; then echo "Please make sure you have set PLATFORMSH_CLI_TOKEN in ~/.ddev/global_config.yaml" && exit 1; fi
+
+db_pull_command:
+  command: |
+    #set -x   # You can enable bash debugging output by uncommenting
+    set -eu -o pipefail
+    ls /var/www/html/.ddev >/dev/null # This just refreshes stale NFS if possible
+    platform db:dump --yes --gzip --file=/var/www/html/.ddev/.downloads/db.sql.gz --project="${PLATFORM_PROJECT}" --environment="${PLATFORM_ENVIRONMENT}"
+
+files_import_command:
+  command: |
+    #set -x   # You can enable bash debugging output by uncommenting
+    set -eu -o pipefail
+    # Use $PLATFORM_MOUNTS if it exists to get list of mounts to download, otherwise just web/sites/default/files (drupal)
+    declare -a mounts=(${PLATFORM_MOUNTS:-/web/sites/default/files})
+    for m in ${mounts[@]}; do
+      platform mount:download --yes --quiet --project="${PLATFORM_PROJECT}" --environment="${PLATFORM_ENVIRONMENT}" --mount=${m} --target=/var/www/html${m}
+    done
```

Comment:
```suggestion
    platform mount:download --yes --quiet --project="${PLATFORM_PROJECT}" --environment="${PLATFORM_ENVIRONMENT}" --all --target=/var/www/html
```

Reply:
I tried --all, but it doesn't work. We don't want all the mounts to go into /var/www/html, which is what happens with your suggestion.

Reply:
They shouldn't be in a public directory at least, but mounts are meant to be relative to the `$PLATFORM_APP_DIR` which is set as `/var/www/html` above.

Reply:
Ah, maybe I just got baffled by the prompts when I tried it. I'll give that a try again.

Reply:
Made this change and tested it, thanks. I did just get surprised by the prompts; and didn't know that the mounts were all meant to be relative, thought they could be absolute.